### DISCUSSION TITLE
posts/627

### DISCUSSION THREAD

**Discussion by giscus** [kakaotech/comments#7] [Announcements]
# posts/627

개요
안녕하세요, 카카오 FE플랫폼 조직에서 광고SDK 업무를 담당했던 이브입니다...

https://tech.kakao.com/posts/627

**Comment by youngjewoo** [comment]
정말 재밌네요

**Comment by jinkwon** [comment]
https://www.npmjs.com/package/fast-average-color 도 참고해 보시면 도움이 될거 같아요.

**Comment by baesku789** [comment]
아이폰se2인데 사파리에사 글이 안 보이네요…
